src/core/dag/StochasticNode.h

```diff
@@ -482,6 +483,23 @@ double RevBayesCore::StochasticNode<valueType>::getLnProbabilityRatio( void )
 }
 
 
+template<class valueType>
+double RevBayesCore::StochasticNode<valueType>::getPrevLnProbability( void ) const
+{
+    // 1. If the node is not affected/touched, then the probability is the same for the current and previous state.
+    if (not stored_ln_prob)
+        throw RbException()<<"getPrevLnProbability: no previous probability!";
```

Comment:
The comment here seems to contradict the error message. Surely having no previous probability is different from the previous probability being equal to the current probability?

Reply:
Yeah, I forgot to modify the comment after copy/pasting the code.  Fixed.